src/azure-cli/azure/cli/command_modules/azure-cli-netappfiles/azure/cli/command_modules/__init__.py

```diff
@@ -0,0 +1,6 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
```

Comment:
These should not exist. There shouldn't be any directories in your `netappfiles` directory except a test folder.

Reply:
Yes. For some reason the deleted files did not merge so I had to manually delete in github. The folders then are not visible. But I no longer see folders or init files. Do you see these in github? I think I need to clone again and push to remove these.

Reply:
Try searching for them in the file explorer. They may not appear in Visual Studio because of the proj file structure.

Reply:
I see now what has happened. This whole azure-cli-netappfiles was wrongly named and I had already changed it to just netappfiles. For some reason the old one also remained.